js/forum/src/main.js

```diff
@@ -9,8 +9,10 @@ import EmojiAreaButton from 'clarkwinkelmann/emojionearea/components/EmojiAreaBu
 
 app.initializers.add('clarkwinkelmann-emojionearea', () => {
     extend(TextEditor.prototype, 'controlItems', function (items) {
-        var emojiButton = new EmojiAreaButton;
-        emojiButton.textEditor = this;
+        const emojiButton = new EmojiAreaButton({
```

Comment:
I saw the use of `new` + direct use in other plugins, and I saw the `props` constructor attribute in the core https://github.com/flarum/core/blob/master/js/lib/Component.js#L37 :eyes: 

And it definitely works !